### DISCUSSION TITLE
Are you droping the project?

### DISCUSSION THREAD

**Discussion by asynchronizd** [KRTirtho/fl-query#29] [Q&A]

**Comment by KRTirtho** [comment]
No, it's being actively maintained. Currently, version 1 is in progress & so commits are being added on a new branch
It had a new alpha release only more than a week ago

So in summary no. I'm sporadically working on it. And it's actually used in multiple projects of mine so no plan for dropping the project

Expect the completely stable v1 release around October when docs & devtools are done
